spark-operator-docs/build.gradle

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+ext {
+  javaMainClass = "org.apache.spark.k8s.operator.utils.ConfOptionDocGenerator"
+  docsPath = System.getProperty("user.dir") + "/docs"
+}
+
+dependencies {
+  implementation project(":spark-operator")
+  implementation("org.projectlombok:lombok:$lombokVersion")
```

Comment:
To @jiangzho , it seems that your repository is outdated a little.

Apache Spark Kubernetes Operator follows the Gradle Version Catalog. Please rebase your repository and refer the following commit.
- https://github.com/apache/spark-kubernetes-operator/pull/109